### PR TITLE
Created and added my self composed Music

### PR DESCRIPTION
**Pull request by saarthakchats**
Added on Basilisk tile. Also created a folder of my name in which I added all my created sounds and also added my license to that folder.

### CODE REVIEW

**Comment by saarthakchats** [comment]
screenshot 57
Look at the volume button on top.

http://rawgit.com/saarthakchats/labyrinth/master/index.html

**Comment by saarthakchats** [comment]
@niccokunzmann

**Comment by niccokunzmann** [comment]
1. audio/background/yashkumarverma.github.io/mybits.mp3 - what is this for?
2. audio/saarthakchats/saarthakchats.license should be renamed, see the documentation
3. Please use the public domain license for that, then

File: js/tiles.js [hunk 1]
```diff
@@ -862,7 +863,7 @@ const door = {
             this.ground = this.createImage("tiles/rooms/floor/Forbidden.svg");
         },
         visit: function() {
-            playAudio('creepy.mp3', 1);
+            playAudio('bensound.com/creepy.mp3', 1);
```
Comment (niccokunzmann) [inline js/tiles.js]: thanks for taking the time to correct these

**Comment by niccokunzmann** [comment]
@saarthakchats What do you think?

**Comment by saarthakchats** [comment]
@niccokunzmann 
Removed that irrelevant sound file, what should I change with the license file?

File: js/tiles.js (hunk 1 above)
Comment (saarthakchats) [reply js/tiles.js]: Anytime!

**Comment by niccokunzmann** [comment]
Please read https://github.com/fossasia/labyrinth#how-to-add-music and ask again.

**Comment by saarthakchats** [comment]
@niccokunzmann 
Separate licenses for both sounds or one readMe and one license file. What should I do?

**Comment by saarthakchats** [comment]
@niccokunzmann 
Is it fine now?

**Comment by niccokunzmann** [comment]
yes! Thanks!

**Comment by saarthakchats** [comment]
Thanks...So are we merging?

**Comment by saarthakchats** [comment]
Oh cool.. Thanks

**Pull request merged**